pages/common/idnits.md

```diff
@@ -0,0 +1,25 @@
+# idnits
+
+> Check internet-drafts for submission nits.
+> Looks for violations of Section 2.1 and 2.2 of the requirements listed on <https://www.ietf.org/id-info/checklist>.
+> More information: <https://tools.ietf.org/tools/idnits/>.
+
+- Check a file for nits:
+
+`idnits {{path/to/file.txt}}`
+
+- Count nits without displaying them:
+
+`idnits --nitcount {{path/to/file.txt}}`
+
+- Show extra information about offending lines:
+
+`idnits --verbose {{path/to/file.txt}}`
+
+- Expect the specified year in the boilerplate instead of the current year:
+
+`idnits --year {{2021}} {{path/to/file.txt}}`
+
+- Assume the document is of the specified status:
```

Comment:
Typo?
```suggestion
- Assume the document is of the specified type:
```

Reply:
No, not exactly.  I think "type" is also correct here, but they're also statuses of the documents.  The full statuses are—
- O → Obsolete
- U → Unknown state
- E → Experimental
- I → Informational
- B → Best Current Practice
- P → Proposed Standard
- D → Draft Standard
- S → Full Standard
- H → Historic
- N → Not Issued

Reply:
Ah, I see. The example there though only has a single named argument `--doctype` though? I'm confused.

Reply:
Yes, `--doctype` specifies the status of the document.  The flag itself says `type`, but _everywhere_ else it's referred to as "status".

`idnits file.txt --doctype bcp` would check `file.txt` for nits, assuming `file.txt` is a "Best Current Practice" rfc draft.

Reply:
Oh, that's really weird! Let's keep this as-is though, but I'm explicitly not resolving this conversation for future reference.